docs/modules/guides/nav.adoc

```diff
@@ -1,5 +1,6 @@
 * Help & Guides
 ** Updating to New Releases
+***xref:api-migration-guide-v25x-to-v30.adoc[]
```

Comment:
There seems to be a blank missing here.
Without that the navigation item "Updating to New Releases" is missing, or has the title "v3.0.x migration guide" instead.

I just realized I did not write anything about the changes to the extensions yet. I'll add that once this is merged.

Reply:
To clarify what I mean, the navigation bar looks like this atm:
<img width="271" alt="Bildschirmfoto 2023-05-29 um 11 24 23" src="https://github.com/asciidoctor/asciidoctorj/assets/1163662/8a933ed5-f88b-4418-a482-8fda43a6acae">

Reply:
Thanks for the fix in the other PR :bow: